### PR TITLE
Updates to About Us, Project Details, and Dashboard

### PR DESCRIPTION
**Pull request by super-ming**
I have a few updates to About Us and Project components. I made a separate project card component so it can be reused for any page. I added a dashboard edit route and component for the profile edit form.

### CODE REVIEW

File: front-end/src/components/auth-redirect/auth-redirect.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ class AuthRedirect extends React.Component {
 
     // default for new user sign-up
     if (!token && path === routes.SIGNUP_FRONTEND) {
-      sendTo = routes.SIGNUP_FRONTEND;
+      console.log(this.props);
```
Comment (Richard-outreach) [inline front-end/src/components/auth-redirect/auth-redirect.js]: remove.

File: front-end/src/components/dashboard/business-profile-edit.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      formValid = false;
+      errors['existingSite'] ='Please enter a valid url.';
+    }
+
+    this.setState({
+        errors: errors
+      });
+    return formValid;
+
+    console.log(fields);
```
Comment (Richard-outreach) [inline front-end/src/components/dashboard/business-profile-edit.js]: same. You get the point.

File: front-end/src/components/auth-redirect/auth-redirect.js (hunk 1 above)
Comment (super-ming) [reply front-end/src/components/auth-redirect/auth-redirect.js]: Removed

File: front-end/src/components/dashboard/business-profile-edit.js (hunk 2 above)
Comment (super-ming) [reply front-end/src/components/dashboard/business-profile-edit.js]: Removed

**Pull request merged**